### PR TITLE
Math-Magician Pages

### PR DESCRIPTION
**Pull request by kinginthenorthcodez**
## Math-Magician Pages
- Create a website consisting of 3 pages: Home, Calculator, and Quote.
- The general layout should match these wireframes:
- Add Home page
- Add Calculator page
- Add Quote page
- Use React basic styling
- For every page, you should create a route.
- Make sure that navigation links for all 3 routes are displayed on each page.
- Use [React Router](https://reactrouter.com/) to make the routes work.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -55,3 +55,84 @@
...
+
+.header .nav {
+  display: flex;
+  width: 100%;
+  max-width: 950px;
+  height: 90%;
+  justify-content: space-between;
+  align-self: center;
+  margin: 0 auto;
+}
```
Comment (AmaduKamara) [inline src/index.css]: - [ ] Kindly remove the `max-width` from these styles as it's not making your design match with the wireframe given.

**Your design**
image

**The Wireframe**
image

- [ ] Kindly add `padding: 0 40px;` to the .header .nav styles and see how things look for the header. 👍 
Adding the padding and removing the max-width will give you a design that looks like the screenshot below.

image

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi @kinginthenorthcodez,

AlonzoLeroneThisCloseGIF (2)

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ No linter error
✅ Professional README
✅ Good descriptive PR

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/index.css (hunk 1 above)
Comment (kinginthenorthcodez) [reply src/index.css]: Removed max-width and added suggested change

**Comment by kinginthenorthcodez** [comment]
@AmaduKamara thanks for the suggestion 👍🏽

**Review by roshan-bajgain — APPROVED** [review]
Hi @kinginthenorthcodez, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️
- Your project is awesome. ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kinginthenorthcodez** [comment]
Merge #5  into development

**Pull request merged**